### PR TITLE
Blazemod Mininuke_mod update

### PR DESCRIPTION
**Pull request by TheRafters**
This reverts the addition from  #15446, that PR was overriding the mods original override present in blaze_other.json, and while it did fix the immediate problem of not being able to fire core mininuke_mods, it broke the intention of blazemod, which was to allow the mininuke launcher to fire several different kinds of ammunition using the h_projectile ammo_type.

Instead, I've overridden the core games mininuke_mod, changing it's ammo type to "h_projectile" and removed the "h_projectile" item from blazemod, which was broken anyway as it did nothing when fired. This restores the intention of the mod, as now it is possible to load and fire any "h_projectile" ammo_type added by the mod and defaults to the mininuke_mod. As a bonus, there is now only 1 mininuke_mod in the item list. I've tested these in my own game and it appears to be functioning correctly and as intended now.

Left: Reloading the mininuke launcher with all it's goodies.
Top Right: Mininuke_mod's stats, listed as a heavy projectile
Bottom Right: Mininuke Launcher's stats, loads heavy projectiles now. Also, only one in the item list.
https://i.imgur.com/rrftESZ.png

Bonus Destruction Photo:
https://i.imgur.com/JL6YPId.png

P.S. Whoever came up with that sound effect for "Your ears ring!" is a sadist...
P.P.S. Sorry Nymall, no offense intended.

### CODE REVIEW

File: data/mods/blazemod/blaze_override.json [hunk 1]
```diff
@@ -417,6 +417,28 @@
...
     },{
+        "type" : "AMMO",
+        "id" : "mininuke_mod",
+        "price" : 180000,
+        "name" : "modified mininuke",
+        "symbol" : "*",
+        "color" : "green",
+        "description" : "A heavy modified handheld nuclear device.  Intended to be fired from a specialized launcher, its case has been cut down and it has been rigged to explode on impact.  While it is now somewhat lighter than before, it can no longer be triggered manually.",
+        "material" : "plastic",
+        "flags": ["LEAK_RAD"],
```
Comment (Coolthulhu) [inline data/mods/blazemod/blaze_override.json]: Does "LEAK_RAD" alone do anything?

**Comment by Malkeus** [comment]
Unknown, it's copied straight out of core files though.

**Comment by Malkeus** [comment]
I searched the entire repository, it's a tag that only appears in a few .json files and doesn't appear to be referenced in the code at all. Remove it?

**Comment by Coolthulhu** [comment]
Yeah.

Does the `h_projectile` item still exist in the game? Removing it just like that will make it cause debugmsgs for those who have it in their saves.

**Comment by Malkeus** [comment]
no, i took it out completely. I could make an obsolete.json similar to the one we have in core and move it there. I'm honestly not sure how you were supposed to create it, there's no recipe for it nor a place in the item lists iirc. If it never spawned, does it need to be available still?

**Pull request merged**